validmind/tests/__init__.py

```diff
@@ -253,67 +256,71 @@ def list_tests(filter=None, task=None, tags=None, pretty=True, truncate=True):
     return tests
 
 
-def load_test(test_id, reload=False):  # noqa: C901
-    # Extract the test ID extension from the actual test ID when loading
-    # the test class. This enables us to generate multiple results for
-    # the same tests within the document. For instance, consider the
-    # test ID "validmind.data_validation.ClassImbalance:data_id_1,"
```

Comment:
Is there any reasons to remove this comments unless you are handling at different place?  The `result_id` specific comment is useful.

Reply:
Nope - that was an oversight, thanks for catching. I added a docstring to the function but I think we should move the test id parsing and complexity to a class eventually.